### PR TITLE
No need for layers.Rescaling(1. / 255.) and layers.Normalization(axis=bn_axis) in efficientnet.py

### PR DESCRIPTION
**Pull request by joonb14**
in tensorflow/tensorflow/python/keras/applications/efficientnet.py
at line 316,317 there is 

x = layers.Rescaling(1. / 255.)(x)
x = layers.Normalization(axis=bn_axis)(x)

We normally put normalized data as input, and other networks(like mobilenet and resnet) other than efficientnet don't have such layers.
I think these two layers should be removed.

### CODE REVIEW

**Comment by joonb14** [comment]
@googlebot I signed it!

**Comment by joonb14** [comment]
@googlebot I fixed it.

**Comment by joonb14** [comment]
@googlebot I fixed it.
